### ISSUE TITLE
botui.action.select  is not a function

### ISSUE THREAD

**Issue by DanielBaron86** [botui/botui#55]
The example for select trows and error

```
botui.action.select({
    action: {
        placeholder : "Select Language",
        value: 'TR,EN', // Selected value or Selected Array object. Example: [{value: "TR", text : "Türkçe" },{value: "EN", text : "English" }]
        multipleselect : true, // Default: false
        options : [
            {value: "EN", text : "English" },
            {value: "ES", text : "Español" },
            {value: "TR", text : "Türkçe" },
            {value: "DE", text : "Deutsch" },
            {value: "FR", text : "Français" },
            {value: "IT", text : "Italiano" },
        ],
        button: {
            icon: 'check',
            label: 'OK'
        }
    }
})
TypeError: botui.action.select is not a function
```

**Comment by mihi-r** [comment]
It's a function that was recently added. You must be using an old clone of the repo. Update your files with the recent updates and it should work fine.

**Comment by drbarto** [comment]
In case others have the same problem: the current (0.3.4) npm release is broken, i.e. contains older code. I installed the current HEAD instead, which worked fine.

**Comment by moinism** [comment]
@DanielBaron86 Apologies for the delay. Fixed in `0.3.5`.
